.cargo/audit.toml

```diff
@@ -0,0 +1,2 @@
+[advisories]
+ignore = ["RUSTSEC-2020-0071"]
```

Comment:
why is that necessary?

Reply:
there is a feature in `chrono` called `time` that is deprecated. The latest version of `chrono` removes it completely but its not fully released yet.
When its active it has an issue that a crate that gets brought in can segfault under certain circumstances.
We don't need that feature but unfortunately its being brought in transitively by `sqlxmq` so we cannot turn it off.
As we are not using any code paths from that feature it shouldn't be an issue and once a chrono crate gets its newest release it shouldn't be an issue anyway.